testutils/vault.go

```diff
@@ -0,0 +1,64 @@
+// +build integration
+// +build vault
```

Comment:
TIL to require both tags go expects separate lines for logical AND, and if on the same line it uses the logical OR operator.